pull_request_template.md

```diff
@@ -1,4 +1,25 @@
-Please go the the `Preview` tab and select the appropriate sub-template:
+Related task: [link to clickup]
 
-* [A template](?expand=1&template=a.md)
-* [B template](?expand=1&template=b.md)
\ No newline at end of file
+# Fixes, Changes, Additions, Removals (What)
+- What was changed from a technical perspective?
+
+# Implementation Reasoning (Why)
+- Why did you implement things the way that you did?
+
+# Impacted Areas (Where)
+- Highlight areas of code that are impacted, especially things that may not be obvious. 
+- Call out how code may impact other devs working on the project.
+- Related branches/pull requests.
+
+# Additional Steps (How, When)
+- Database migrations.
+- Console commands.
+- Team or client follow up.
+- Dependency management commands.
+
+# Screenshot(s)
+- Example of what the feature looks like.
+
+# Bug context and root cause
+- Link to bug tracker, ex: Sentry.
+- What caused the bug.
```

Comment:
I'd move "Bug context and root cause" as a sub-section under "Fixes, Changes, Additions, Removals (What)". I think it has a better flow since it moves into implementation next

```suggestion
# Fixes, Changes, Additions, Removals (What)
- What was changed from a technical perspective?

## Bug context and root cause
- Link to bug tracker, ex: Sentry.
- What caused the bug.

# Implementation Reasoning (Why)
- Why did you implement things the way that you did?

# Impacted Areas (Where)
- Highlight areas of code that are impacted, especially things that may not be obvious. 
- Call out how code may impact other devs working on the project.
- Related branches/pull requests.

# Additional Steps (How, When)
- Database migrations.
- Console commands.
- Team or client follow up.
- Dependency management commands.

# Screenshot(s)
- Example of what the feature looks like.
```